WireDomain/Sources/WireDomain/Event Processing/ConversationEventProcessor/ConversationEventProcessor.swift

```diff
@@ -54,13 +53,14 @@ struct ConversationEventProcessor {
     func processEvent(_ event: ConversationEvent) async throws {
         switch event {
         case let .accessUpdate(event):
-            try await accessUpdateEventProcessor.processEvent(event)
+            await accessUpdateEventProcessor.processEvent(event)
 
-        case let .codeUpdate(event):
-            try await codeUpdateEventProcessor.processEvent(event)
+        case .codeUpdate:
+            break // Event is not currently processed instead we fetch guest link on demand directly from API, see
+            // `CreateConversationGuestLinkUseCase` and `CreateConversationGuestLinkActionHandler`
```

Comment:
nitpick: it'd be nice to put comments on the line above.

Reply:
fixed in https://github.com/wireapp/wire-ios/pull/2206/commits/c7d60ea7c8b3da610f9f4e41e59f9dc4c57a7c2d